pkg/values/README.md

```diff
@@ -0,0 +1,17 @@
+# About this package
+
+This package is separate from the main module, so it can be imported by the CRE SDK without unnecessary dependencies.
+
+To update the chainlink-protos commit, update the variable chainlinkProtosVersion in installer/pkg/consts.go.
+
+# installer
+Use the installer package to ensure that the chainlink-protos commit always matches the version used in this package.
```

Comment:
@nolag IIUC this is a general tool to generate capability protos, correct? If so, is putting this in the values pkg the right place? Feels like we're overloading the package a bit